### ISSUE TITLE
Seeing a WebDriverError running the benchmarks

### ISSUE THREAD

**Issue by agubler** [krausest/js-framework-benchmark#364]
Seems a release of chromedriver may have impacted running the benchmarks:

```
Benchmark failed { WebDriverError: unknown error: call function result missing 'value'
  (Session info: chrome=65.0.3325.146)
  (Driver info: chromedriver=2.34.522932 (4140ab217e1ca1bec0c4b4d1b148f3361eb3a03e),platform=Mac OS X 10.13.3 x86_64)
    at Object.checkLegacyResponse (webdriver-ts/node_modules/selenium-webdriver/lib/error.js:546:15)
    at parseHttpResponse (webdriver-ts/node_modules/selenium-webdriver/lib/http.js:509:13)
    at doSend.then.response (/webdriver-ts/node_modules/selenium-webdriver/lib/http.js:441:30)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7) name: 'WebDriverError', remoteStacktrace: '' }
```

Although considering the benchmarks are run against Chrome 62 (as stated in the interactive results) this actually might still be okay.

Tested locally and upgrading to chromedriver 2.3.6 resolves the issue with Chrome 65

**Comment by krausest** [comment]
Thanks - fixed in 4095f53538bfa6ee9de5991d88bbbecec7c11f5a
Strange that it worked on my MacBook, would be nice if you could check whether it works now for you.

**Comment by agubler** [comment]
Yup perfect, thanks
